GitHub-case-study.md

```diff
@@ -0,0 +1,132 @@
+
+## Features
+
+This is GitHub key features:
+
+- **Version Control**: version control is also known as revision control, source control, or source code management. It allows developers to monitor changes to their code and they can revert to previous versions if needed.
+
+- **Team-Collaboration**: Users can work together on the same project, inspect each others code, raise issues and make plans for the project.
+
+- **Branches**: A user can start a new branch either from the main branch or a different branch. This allows the user to write new code for the project without adding it to the project itself. When new code is written on a new branch it can be merged into the main project. Usually users make a pull request before merging.
+  This system enables users to develop code on different versions of a project at the same time.
+
+- **Pull requests**: When a user have made changes to a project's code on a separate branch, he can make a pull request. This is an important step in quality control where other developers can inspect the changes in the code. They do this to make sure that the code is bug free and has no issues. When it is determined that the code is safe it can be merged from this branch to it's parent branch or to the main branch.
+
+- **Issues**: GitHub Issues can be used to track ideas, feedback, tasks, or bugs for work on GitHub. Issues can be an important part of development as bugs and problems with the code arises developers can raise an issue and also assign the issue to specific developers for fixing.
+
+- **Merging**: Is adding new code to a project from a separate branch. usually this is done after a pull request.
+
+- **Integration**: GitHub allows for developers to use other programs in combination with github. examples are: continuous integration systems, tools for project management and several IDEs.
+
+- **GitHub Actions**: Is a feature where you can add automated tasks to a project. The automated tasks can either be run manually or be set to run at specific points of development such as when a pull request is made. GitHub actions is very flexible and allows for lots of different programming languages and tools to be used with it.
+  examples of GitHub actions usage: deploy to static pages, end to end testing and unit testing.
+
+- **GitHub Pages**: Can be used to host a web project directly from GitHub. It is an easy way for developers to host a project and/or share details for a project.
+
+- **GitHub Enterprise**: Allows businesses to run their github projects on their own servers with additional security and access to more features like GitHub Enterprise API.
+
+## Strengths
+
+- **Web-based platform/ease of access**: Being web based i see as a plus. New users can easily create an account especially when they can sign in via google.
```

Comment:
No `i` in technical writing. Rather use phrases such as `It is this author's view`, or `we view` etc.